### DISCUSSION TITLE
GitHub Pages -- Successful Deployment, But Page unavailable. ERR_CONNECTION_RESET

### DISCUSSION THREAD

**Discussion by walakaka77** [community/community#103210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Dear all,

# Description

Previously created a thread for this but the issue was resolved. But resurfacing this because the issue appeared again:
1. Created a GitHub Page using Jekyll
2. Pushed the GitHub Page to the Repo -- named `walakaka77.github.io`, following the following naming convention `username.github.io`
3. Deployment was successful
4. But on trying to access the page over the internet, receiving a `ERR_CONNECTION_RESET`

Screenshots of configurations are available below, for your reference.

## Screenshots of ERR_CONNECTION_RESET

![image](https://github.com/community/community/assets/81464351/c94b937f-7035-4d6f-b57d-c699b523b8e9)

## Screenshot of Successful Deployment
![image](https://github.com/community/community/assets/81464351/1495ad5d-2923-463a-810d-0cc2eb9d88c2)

## Screenshot of GitHub Pages Settings
_Note: Am using GitHub Actions_

![image](https://github.com/community/community/assets/81464351/d3a648c0-d0f7-42bb-89e1-8d98eb856069)

## Location of GitHub Action YAML Codeblock

Can see the [GitHub Action Workflows YAML Here!](https://github.com/walakaka77/walakaka77.github.io/tree/main/.github/workflows)

**Comment by DJLink** [comment]
Seems like it sorted itself since I can see it on Firefox now, so it's live and working. 


![image](https://github.com/community/community/assets/423769/62f542da-4e2e-4956-a857-43a0e488f823)
